examples/sushi/linter/user.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import typing as t
+
+from sqlmesh.core.linter.rule import Rule, RuleViolation
+from sqlmesh.core.model import Model
+
+
+class NoMissingOwner(Rule):
+    def check(self, model: Model) -> t.Optional[RuleViolation]:
+        return RuleViolation(rule=self, model=model) if not model.owner else None
+
+    @property
+    def summary(self) -> str:
```

Comment:
should we instead make this a docstring? on the class

class NoMissingOwner:
  """my sumamry"""